samples/dynamodb-geo-server/pom.xml

```diff
@@ -103,6 +103,37 @@
 						</webResources>
 					</configuration>
 				</plugin>
+				<plugin>
+					<groupId>org.apache.tomcat.maven</groupId>
+					<artifactId>tomcat7-maven-plugin</artifactId>
+					<version>2.2</version>
+					<configuration>
+						<!-- http port -->
+						<port>9090</port>
+						<!-- application path always starts with /-->
+						<path>/</path>
+						<!-- optional path to a context file -->
+						<contextFile>${tomcatContextXml}</contextFile>
+						<!-- optional system propoerties you want to add -->
+                        <systemProperties>
+							<!--
+                            <AWS_ACCESS_KEY_ID>AKIAJBG2H2TJ5DSDOS6A</AWS_ACCESS_KEY_ID>
```

Comment:
Yep, the access key id gets used to name the database file so you can run multiple tests concurrently without them interfering, but the signature isn't actually validated so it doesn't matter if you use an actual access key. Thanks for the contribution, we'll take a look tomorrow!